recipe/meta.yaml

```diff
@@ -26,10 +26,13 @@ requirements:
     build:
         - python  # [win]
         - msinttypes  # [win]
+    #run:
+        #- libgcc  # [unix]
```

Comment:
No, that's a good question. IIRC we will include a `libgcc` package. However, that package will just copy over the libraries from the `gcc` that we build (and build other things with). See this PR ( https://github.com/conda-forge/staged-recipes/pull/135 ) for details about the proposal for doing this.